services/py-api/src/database/model/admin/team_member_model.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Dict
+
+from src.database.model.admin.base_admin_model import AdminBaseModel, AdminUpdateParams
+
+
+@dataclass(kw_only=True)
+class TeamMember(AdminBaseModel):
```

Comment:
What is the Idea of this Data model, and how does it differ from the existing `Participant` model?